sdk/endpoint/endpoint.go

```diff
@@ -44,15 +44,27 @@ type NsmEndpoint interface {
 	Delete() error
 }
 
+// NsmEndpointRegistration stores parameters that NsmEndpoint passes to the
+// RegisterNSE gRPC call.
+type NsmEndpointRegistration struct {
```

Comment:
In general, it is a nice solution for me:) 
```suggestion
type EndpointRegistration struct {
```

Reply:
The linter doesn't like repetition (`endpoint.EndpointRegistration`), so I'll name it `endpoint.Registration` then.

Reply:
endpoint.Registration sounds better :)